### DISCUSSION TITLE
Can you update your answers and template with the same vcs ref?

### DISCUSSION THREAD

**Discussion by coretl** [copier-org/copier#1469] [Q&A] [answered]
We have a template that uses the GitHub organisations [throughout the project](https://github.com/DiamondLightSource/python_copier_template/blob/29c2f957c44b6e310521453ca54e99371ad7f014/template/README.rst.jinja#L43). When we transfer between GitHub organisations we would like to run `copier update` and have it ask the questions again but using the **same** vcs-ref as recorded in `.copier-answers.yml`. The only way I can see to do this extracting `_commit`  from `.copier-answers.yml` by hand and supplying it as a `--vcs-ref` argument to `copier update`. Is there a better way to do this?

**Comment by yajo** [comment, marked as answer]
Currently all actions (copy, recopy, update) default to the last PEP440-sorted tag available in the repo.

We'd need something like a new ˋ--keep-vcs-refˋ  option. FWIW I have no plans on developing it, because until today the current workaround that you specified has been good enough for me. 

But if you want you can open an issue, so we can track it and maybe someone wants to contribute it.
